src/app/pages/Swap/modals/SwapSettings/index.tsx

```diff
@@ -41,16 +42,16 @@ const SwapSettingsModal = memo<SelectTokenModalProps>(({ onSubmit, opened, onReq
 
   const handleOptionChange = (option: (typeof options)[number]) => {
```

Comment:
Please introduce and use a type like `SlippageOption` to simplify type definitions like this one